draft-ietf-rats-corim.md

```diff
@@ -1066,6 +1066,182 @@ The following describes each member of the `concise-bom-tag` map.
+
+We make no assumptions on the specific shape of the Attestation Result, except for its optional ability to include Evidence from the attestor and Endorsed Values that the Verifier has been able to infer from Evidence and the Appraisal Context.
+
+> A CoRIM profile MUST describe:
+>
+> * How Reference Values are conveyed (e.g., using Reference Value triples, or CoSWID tags)
+> * How Reference Values are associated with the Target Environment
+> * How the Target Environment is identified in Evidence
+> * How measurements from Evidence are matched against Reference Values
+
+> A CoRIM profile MUST describe:
+>
+> * How Endorsed Values are conveyed (e.g., using Endorsed Value triples, or Device Identity triples)
+> * How Endorsed Values are associated with the Target and Attesting Environments
+> * How the Target and Attesting Environments are identified in Evidence
+
+The sections below describe the algorithm used to compare CoMID tags.
+
+### Comparing and processing CoMID tags
+
+There can be multiple passes of the CoMID evidence appraisal process.
+
+In each pass, each potential CoMID tag in the Appraisal Context is compared against the accepted claims set (which has been initialised with Evidence from the attestor(s) ). If the reference values in the CoMID tag match the Accepted Claims Set then Endorsements and other values from the tag are added to the Accepted Claims Set.
+
+If any tags matched in a pass through the CoMID appraisal process then another pass is performed. Tags which matched in a previous pass are ignored in later passes of the same appraisal.
+
+A later pass may match Reference Values from the CoMID against Endorsements from matched tags in an earlier pass, this allows CoRIM authors to create hierarchical descriptions of a system.
+
+### Matching Evidence against Reference Values
+
+This section describes the process performed by the verifier to determine whether a candidate CoMID tag matches the Accepted Claims Set. If a tag does not match the accepted claims set then it is silently ignored for this pass, processing continues with the remaining candidate tags.
+
+The verifier enumerates over the reference values in the CoMID tag. If the Accepted Claims Set does not contain an entry with the same environment-map as the reference claim then the tag does not match. Comparison of environment-map is performed using a binary comparison. Verifiers SHOULD convert environment-map to the canonical format before performing the binary comparison.
+
+The verifer locates the corresponding measurement-values-map in the reference value and the Accepted Claims Set. The verifier enumerates over the entries in the reference value measurement-values-map. If the Accepted Claims set measurement-values-map does not have an entry with the same key as the reference value measurement-values-map, or if the entries do not match, then the tag does not match.
```

Comment:
Can we describe the method using more generic terminology such as "corresponding measurements, (e.g., `measurement-values-map`)" so that the specific corim statements are exemplary.